other/res/restrict-node-label-creation/03-check.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- script: |
```

Comment:
Same sort of test here. Make sure you can set some other label aside from `foo`.

Reply:
Added.